### PR TITLE
Automatically add events from Meetup

### LINKED ISSUE #66: [Feature Request]: Automatically add events from Meetup
### Description

As an admin of spokanetech.org, I would like to not have to manually add all events for our tech groups. Instead, I would like a periodic background job to scrape / pull via API new events and add them to our events list.

### Problem

_No response_

### Additional context

The solution should be flexible enough to support scraping different websites and/or retrieving events via different APIs.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by joeriddles**
## Pull Request

**Description:**
- Add automatic scraping of Tech Groups that use Meetup.com as their home page to update the events on SpokaneTech.org

**Related Issues:**
- closes #66

**Checklist:**
- [x] All tests pass.
- [x] Code follows the project's coding standards.
- [x] Documentation has been updated.

### CODE REVIEW

**Review by KGB33 — COMMENTED** [review]
Is there any way to run the scrapers locally? 
I couldn't get them to run after playing around in the Admin panel.

File: src/web/tests/test_scrapers.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+
+class TestMeetupHomepageScraper(TestCase):
+
+    @freezegun.freeze_time("2024-03-18")
+    @responses.activate
+    def test_scraper_with_json(self):
+        fin = open(pathlib.Path(__file__).parent / "data" / "meetup-homepage-with-json.html")
+        body = fin.read()
+        fin.close()
```
[the remark this answers is not in the document]
Comment (joeriddles) [reply src/web/tests/test_scrapers.py:20]: Nope! Will update

**Comment by joeriddles** [comment]
> Is there any way to run the scrapers locally?
> I couldn't get them to run after playing around in the Admin panel.

I need to update the docs to explain how to run it locally. Basically you need to run redis or rabbitmq locally, likely in a Docker container, and update the CELERY_BROKER_URL to point to the broker (redis / rabbitmq).

**Comment by joeriddles** [comment]
@KGB33  I added a Celery section to the Contributing docs. Let me know if those steps work for you!

**Review by KGB33 — APPROVED** [review]
LGTM. 

Thanks for adding the celery docs, I was able to get everything working locally.

**Pull request merged**